app/src/main/java/org/jellyfin/mobile/utils/ApiExtensions.kt

```diff
@@ -0,0 +1,27 @@
+package org.jellyfin.mobile.utils
+
+import org.jellyfin.apiclient.interaction.ApiClient
+import org.jellyfin.apiclient.interaction.Response
+import org.jellyfin.apiclient.model.system.PublicSystemInfo
+import timber.log.Timber
+import kotlin.coroutines.Continuation
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+
+val PRODUCT_NAME_SUPPORTED_SINCE: IntArray = intArrayOf(10, 3)
```

Comment:
```suggestion
val PRODUCT_NAME_SUPPORTED_SINCE: Pair<Int, Int> = 10 to 3
```

Reply:
You really don't like arrays, do you? :stuck_out_tongue_closed_eyes:

Reply:
No need for an array when using 2 values /shrug

Reply:
And you're right, for some reason I never use arrays. Always lists/sets idk why